bftengine/src/communication/TlsTCPCommunication.cpp

```diff
@@ -35,11 +35,13 @@
 #include "openssl/ssl.h"
 #include <openssl/x509.h>
 #include <openssl/x509v3.h>
+#include "Logging.hpp"
 
 class AsyncTlsConnection;
 
 using namespace std;
 using namespace bftEngine;
+using namespace concordlogger;
 using namespace boost::asio;
```

Comment:
will refactor. thanks!

Reply:
I have read a bit about this question - it appears to be asked a lot. Although google's code style impose not to use it, there are few decent c++ experts that don't see any problems with it. Using namespaces makes the code shorter. Anyway, if there is a need to be explicit on which function you really intend to call, one may always use the fully qualified name. I agree that specifically for Asio it may be a good idea to use fully qualified names, but on the other hand I don't see any reason to write std:: everywhere. Of course, in headers we will never write "using namespace".